substrate/dataclass_models.py

```diff
@@ -22,10 +22,6 @@ class ErrorOut:
     """
     A message providing more details about the error.
     """
-    request_id: Optional[str] = None
```

Comment:
hm this dataclass_models.py file seems to be stale and unused, will look into removing